src/styling/PopularList.css

```diff
@@ -0,0 +1,66 @@
+.movie-wrapper {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: flex-end;
```

Comment:
Just a question here...Did you want to present the movies all the way to right? or you want them to be center? 
-Izabel&Etna